Cargo.toml

```diff
@@ -59,7 +59,7 @@ risc0-zkp = { version = "1.2" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_yaml = "0.9"
-guest-set-builder = { git = "https://github.com/risc0/risc0-ethereum", tag = "aggregation-v0.2.1" }
+guest-set-builder = { git = "https://github.com/risc0/risc0-ethereum", tag = "aggregation-v0.2.2" }
```

Comment:
I don't understand, why this points to a special aggregation tag instead of to the general latest version of risc0-ethereum

Reply:
I think the idea was to keep it more independent from the rest of the repo